### ISSUE TITLE
Handles not sliding

### ISSUE THREAD

**Issue by tedp** [angular-slider/ng5-slider#158]
I'm implemented a reactive slider based on the 'Range slider in reactive form' however when I grab the handles and move them the values change but the handles stay where they are.  Is this something you have seen before?  I'm using Angular 6.0.1

**Comment by tedp** [comment]
I seem to have it working if I change to a template form...

**Comment by tedp** [comment]
I could do with some help please when using reactive forms.  I can get the slider to display but when I drag either pointer, although the pointer label updates, the pointer position does not.  Strange thing is that I am unable to re-create this in a sample app so I can't show you the problem.  

I've stepped through the code in the browser and I can see the relevant methods on the SliderElementDirective are being called with new left position values but these are not being applied to the DOM

Can you think of any reason why I might be having such a problem?

**Comment by tedp** [comment]
I found the problem.  It was because in my real app we have a hierarchy of view components all with change detection strategy of onPush.  If I listen for value changes to my form object and then for a change detection then the pointers move.

`this.form.valueChanges.subscribe(() => this.cdRef.detectChanges());`

Am I missing anything that would avoid having to do this?

**Comment by tedp** [comment]
Here is a Stackblitz that illustrates the problem.  The top bar uses a reactive slider where as the bottom one uses template based slider.  You can see that the pointers of the top slider do not move but the pointers of the template one does.  Note also that when you move the template slider pointers the reactive slider 'catches up' with it's updates.

https://stackblitz.com/edit/ng5-slider-static-pointers

**Comment by tedp** [comment]
I've added a question on [Stackoverflow](https://stackoverflow.com/questions/58956465/ng5-slider-dragging-pointers-not-moving-when-used-in-a-angular-reactive-form).  Maybe someone will respond there...

**Comment by tedp** [comment]
I've been looking into this further.  I don't pretend to understand all the slider code but adding a `this.changeDetectionRef.markForCheck()` in the `SliderElementDirective` seems to do the trick:

```javascript
  setPosition(pos: number): void {
    this._position = pos;
    if (this._vertical) {
      this.bottom = Math.round(pos) + 'px';
    } else {
      this.left = Math.round(pos) + 'px';
    }
    if (this.changeDetectionRef) {
      this.changeDetectionRef.markForCheck();
    }
  }
```

Is there anyone out there that can comment on this?

**Comment by chriszrc** [comment]
Reactive form support also appears to be broken if you want two slider handles :(

**Comment by piotrdz** [comment]
@tedp Thanks for suggesting a fix. I made a fix along these lines in current master branch and I will make next release as soon as I can.

**Comment by piotrdz** [comment]
Released in v1.2.5. Closing.

**Comment by fzs1994** [comment]
This is still not working in either v1.2.5 or latest. I'm using multiple sliders inside ngFor
